packages/debug/src/browser/editor/debug-hover-widget.ts

```diff
@@ -171,17 +175,43 @@ export class DebugHoverWidget extends SourceTreeWidget implements monaco.editor.
                 this.activate();
             }));
         }
-        if (!await this.hoverSource.evaluate(expression)) {
+        const expression = await this.hoverSource.evaluate(matchingExpression);
+        if (!expression) {
             toFocus.dispose();
             this.hide();
             return;
         }
-        this.editor.getControl().layoutContentWidget(this);
+
+        this.contentNode.hidden = false;
+        this.domNode.classList.remove();
```

Comment:
Does this call do anything? I don't see an API for `.remove()` with no arguments, but I may be missing something.

Reply:
No. It's garbage. Thanks for the note. I fixed it.